includes/class-newspack-listings-featured.php

```diff
@@ -64,14 +64,24 @@ public static function instance() {
 	 * Constructor.
 	 */
 	public function __construct() {
-		register_activation_hook( NEWSPACK_LISTINGS_FILE, [ __CLASS__, 'create_custom_table' ] );
-		add_action( 'plugins_loaded', [ __CLASS__, 'check_update_version' ] );
-		add_action( 'init', [ __CLASS__, 'register_featured_meta' ] );
-		add_action( 'init', [ __CLASS__, 'cron_init' ] );
-		add_action( self::CRON_HOOK, [ __CLASS__, 'check_expired_featured_items' ] );
-		add_filter( 'posts_clauses', [ __CLASS__, 'sort_featured_listings' ], 10, 2 );
-		add_filter( 'post_class', [ __CLASS__, 'add_featured_classes' ] );
-		add_filter( 'newspack_blocks_term_classes', [ __CLASS__, 'add_featured_classes' ] );
+		if ( self::is_active() ) {
```

Comment:
Nit: maybe exit early instead of this big conditional block?

Reply:
Implemented in d618b29.